target/linux/ath79/base-files/etc/board.d/01_leds

```diff
@@ -168,6 +168,7 @@ tplink,archer-d50-v1)
 	ucidef_set_led_switch "wan_data" "WAN Data" "tp-link:white:internet" "switch0" "0x02" "" "tx rx"
 	ucidef_set_led_switch "wan_link" "WAN Link" "tp-link:white:wan" "switch0" "0x02" "" "link"
 	;;
+tplink,wbs210-v2|\
```

Comment:
Please add the device identifiers so that alphabetic order is maintained (also in all cases below).